### ISSUE TITLE
Macerator breaks all grinding recipes

### ISSUE THREAD

**Issue by bluelightning32** [wqpvs/qptech#10]
I have the MoreMetals mod installed (v1.0.9) along with qptech1.7.2.zip in VS 1.16.4-rc2. I was processing magnetite ore with the macerator. It correctly added magnetite nuggets into the chest.

However, on the client it looked like a bunch of crushed magnetite also was spilling onto the ground near the chest. Whenever I went to pick up the crushed magnetite, it would disappear without going into my inventory.

![image](https://user-images.githubusercontent.com/21202621/154667022-e54ee936-b2c1-4517-85a4-5e9630f16949.png)

Magnetite ore should have only produced magnetite nuggets. I tried putting the magnetite nuggets back into the macerator, which should have produced crushed magnetite, but only about 1/5 of the expected output was received.

**Comment by bluelightning32** [comment]
I removed all other mods and I was able to reproduce another broken grinding recipe in a single player world. That is a grinding recipe broke with just qptech1.7.2 and VS 1.16.4-rc2.

Repro steps:
1. Setup a quern and try grinding limestone into lime. It works.
2. Setup a qptech macerator and put limestone into it. Only one piece of lime goes into the chest. The rest of the lime starts falling on the ground:
![image](https://user-images.githubusercontent.com/21202621/154834186-b0f4c751-ea1c-4881-86a2-38dc3defc029.png)
3. Pick up the lime from the ground. It does not go into your inventory.
4. Go back to the quern. Take out the completed lime. Put in new limestone if necessary.
5. Now the quern is broken. It consumes limestone but produces 0 sized stacks of lime. It also spills zero sized stacks of lime onto the ground.
![image](https://user-images.githubusercontent.com/21202621/154834248-096a4b87-bfb8-43fb-8cf1-194aa17151e4.png)
